src/components/data/listing/Listing.js

```diff
@@ -90,6 +94,15 @@ function Listing(props) {
         },
     });
 
+    const refreshListing = () => queryCache.refetchQueries(dataPagedListingKey);
+
```

Comment:
I think it will be good do a force refetch here to prevent cached data being returned. See https://github.com/tannerlinsley/react-query#querycacherefetchqueries